pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -1843,3 +1867,39 @@ func getProvisions(c client.Client) []*hivev1.ClusterProvision {
 	}
 	return provisions
 }
+
+func createSyncSetInstanceObj(syncCondType hivev1.SyncConditionType) *hivev1.SyncSetInstance {
+	ssi := &hivev1.SyncSetInstance{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      testSyncsetInstanceName,
+			Namespace: constants.HiveNamespace,
+		},
+	}
+	ssi.Spec.ClusterDeployment.Name = testClusterName
+	ssi.Status = createSyncSetInstanceStatus(syncCondType)
+	return ssi
+}
+
+func createSyncSetInstanceStatus(syncCondType hivev1.SyncConditionType) hivev1.SyncSetInstanceStatus {
+	conditionTime := metav1.NewTime(time.Now())
+	var ssiStatus corev1.ConditionStatus
+	var condType hivev1.SyncConditionType
+	if syncCondType == hivev1.ApplyFailureSyncCondition {
+		ssiStatus = corev1.ConditionFalse
```

Comment:
This status should be `corev1.ConditionTrue`. The condition is true when there is a failure applying the sync set.

Reply:
This is information to me. I was not aware of this.